libr/anal/esil.c

```diff
@@ -2021,10 +2021,10 @@ static bool esil_poke_n(RAnalEsil *esil, int bits) {
 			if (bits == 128) {
 				src2 = r_anal_esil_pop (esil);
 				if (src2 && r_anal_esil_get_parm (esil, src2, &num2)) {
-					r_write_ble (b, num, esil->anal->config->big_endian, 64);
+					r_write_ble (b, num, R_ARCH_CONFIG_IS_BIG_ENDIAN (esil->anal->config), 64);
```

Comment:
just hold `bool be =` once at the begining of the function to make it more readable and avoid to compute this many times